html/janus.js

```diff
@@ -2238,4 +2275,18 @@ function Janus(gatewayCallbacks) {
 			return true;	// Default is true
 		return (trickle === true);
 	}
+
+	function isScreenshare(media) {
+		return (
+			media.video === 'screen' ||
+			media.video === 'window' ||
+			media.video === 'tab' ||
+			(
+				typeof media.video === "object" &&
+				media.length &&
+				(( media.indexOf('screen') + media.indexOf('window') + media.indexOf('tab')) > -3)
```

Comment:
I don't understand this check: `media` is an object, why would you check its `indexOf`?